docs/painless/painless-contexts/painless-runtime-fields-context.asciidoc

```diff
@@ -0,0 +1,218 @@
+[[painless-runtime-fields-context]]
+=== Runtime fields context
+beta::[]
+Use a Painless script to return a calculated value for a
+<<painless-runtime-fields,runtime field>>, defined either in the mappings or as
+part of a query. You can also use runtime fields to override values for indexed
+fields.
```

Comment:
I would say that the link is enough and we don't need to expand on where they are defined and that they can override existing fields?

Reply:
Works for me 👍